config/environments/development.rb

```diff
@@ -28,8 +32,25 @@
   # Do not compress assets
   config.assets.compress = false
 
+  # Generate digests for assets URLs.
+  #
+  # Asset digests allow you to set far-future HTTP expiration dates on all assets,
+  # yet still be able to expire them through the digest params.
+  config.assets.digest = !!ENV["PROFILE"]
+
   # Expands the lines which load the assets
-  config.assets.debug = false
+  #
+  # Setting this to false makes Rails bundle assets into all.js and all.css.
+  #
+  # Disabling asset debugging still requires that assets be compiled for each
+  # request. You can avoid that by precompiling the assets as in production:
+  #
+  #   $ bundle exec rake assets:precompile:primary assets:precompile:nondigest
+  #
+  # You can remove them by simply running:
+  #
+  #   $ bundle exec rake assets:clean
+  config.assets.debug = !ENV["PROFILE"]
```

Comment:
I missed this single !
that means this PR changes everyone's local setup to separate all js files in their dev tools.
I dont like this, search all files in devtools (CTRL SHIFT F) is not so fast as searching in one single file with all the JS.

But if I set PROFILE I get all the other production like settings....
Do you mind if I create a PR to change this line to maybe use another env var like ENV["DEBUG_ASSETS"]?

Reply:
fine for me :+1:

Reply:
Here's the PR:
https://github.com/openfoodfoundation/openfoodnetwork/pull/5344

I found this also makes my local server slower. I think it's because of the hundreds of js files that need to be downloaded and used.